aspnet/mvc/views/partial.rst

```diff
@@ -1,8 +1,128 @@
+
+.. note:: If you're coming from an ASP.NET Web Forms background, partial views are similar to `user controls <https://msdn.microsoft.com/en-us/library/y6wb1a0e.aspx>`_.
+
+When Should I Use Partial Views?
+--------------------------------
+
+Partial views are an effective way of breaking up large views into smaller components, either for reuse or simply to make them easier to manage. If you have a complex page made up of several logical sections, it can be helpful to work with each section as its own partial view. Each section of the page can be viewed in isolation from the rest of the page, and the view for the page itself becomes much simpler since it only contains the overall page structure and calls to render the partial views.
+
+Another common use case for partial views is reusable view content. Common layout elements should be specified in :doc:`layout.cshtml <layout>`, but non-layout reusable content can be encapsulated into partial views, reducing duplication within your views.
+
+.. tip:: Try to follow the `Don't Repeat Yourself Principle <http://deviq.com/don-t-repeat-yourself/>`_ in your views, as well as in your code.
+
+Declaring Partial Views
+-----------------------
+
+Partial views are created identically to regular views. Simply create a new *.cshtml* file and place it somewhere under your *Views* folder. There is no structural difference between a partial view and a regular view - it's just a difference in how they are rendered. If you want, you can have a regular view that is returned as part of controller's ``ViewResult``, and which is also included in other views as a partial view. The main difference between how a view and a partial view are rendered is that partial views do not run *_ViewStart* (while views do - learn more about *_ViewStart* in :doc:`layout`).
+
+Referencing a Partial View
+--------------------------
+
+From within a view page, there are several ways in which you can bring in a partial view. The simplest is to use ``Html.Partial``, which returns an ``IHtmlString`` and can be referenced by simply prefixing the call with ``@``:
+
+.. literalinclude:: partial/sample/src/PartialViewsSample/Views/Home/About.cshtml
+  :lines: 9
+
+You can also render partial views asynchronously using ``PartialAsync``:
+
+.. literalinclude:: partial/sample/src/PartialViewsSample/Views/Home/About.cshtml
+  :lines: 8
+
+A third way to render partial views is to call directly to ``RenderPartial``. This method doesn't return a result, it streams the rendered output directly to the response. Because it doesn't return a result, it must be called within a razor code block. You can also call its async equivalent, as shown here:
```

Comment:
You can render a partial view with :dn:method:`~Microsoft.AspNetCore.Mvc.Rendering.IHtmlHelper.RenderPartial`.